_data/identity.yml

```diff
@@ -133,4 +133,4 @@ websites:
       sms: Yes
       exceptions:
           text: "2FA is only available on paid plans."
-      doc: http://www.securesafe.com/en/faq/security.html
+      doc: http://www.securesafe.com/en/faq/security.html
```

Comment:
As per our `.editorconfig`, it contains `insert_final_newline = true`.
Please respect that.